Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [workspace]
 members = [
```

Comment:
Could we get rid of the formatting changes here? It makes it a bit more difficult to discern actual changes from whitespace changes & we'll likely end up with a bit of ping pong with formatting between PRs. We could consider adopting https://github.com/tamasfe/taplo to format tomls.